va/va.h

```diff
@@ -4041,6 +4047,12 @@ VAStatus vaSyncBuffer(
  * A, V, Y, U are found in bits 31:30, 29:20, 19:10, 9:0 respectively.
  */
 #define VA_FOURCC_Y410          0x30313459
+/** Y412 packed 12-bit YUVA 4:4:4.
+ *
+ * Each pixel is a set of four samples, each of which is a two-byte little-endian value.
+ * The samples are in the order A, V, Y, U.
```

Comment:
@fulinjie how about to define it to X, V, Y U directly, to address concern from ffmpeg community?

Reply:
Yes, community seems to prefer XVYU if Alpha channel is not actually supported.